README.md

```diff
@@ -1,4 +1,4 @@
-# GP2GP Sending Adaptor
+# GP2GP Sending Adaptor [![GP2GP Build Workflow](https://github.com/NHSDigital/integration-adaptor-gp2gp/actions/workflows/build_workflow.yml/badge.svg)](https://github.com/NHSDigital/integration-adaptor-gp2gp/actions/workflows/build_workflow.yml)
```

Comment:
```suggestion
# GP2GP Sending Adaptor 
[![GP2GP Build Workflow](https://github.com/NHSDigital/integration-adaptor-gp2gp/actions/workflows/build_workflow.yml/badge.svg)](https://github.com/NHSDigital/integration-adaptor-gp2gp/actions/workflows/build_workflow.yml)
```

Not precious, just feels strange having the badge in the title!

Reply:
How about we merge it like this for now, and you can review if it still feels strange after a week or so 😄